cpp/src/io/orc/reader_impl.cu

```diff
@@ -336,6 +366,11 @@ rmm::device_buffer reader::impl::decompress_stripe_data(
   bool use_base_stride,
   rmm::cuda_stream_view stream)
 {
+  // For checking whether we decompress successfully
+  hostdevice_vector<bool> any_page_failure(1, stream);
```

Comment:
nit: I think in ORC, the smallest compressed unit is called block. There is no "page" in ORC afaik.

Reply:
renamed to `any_block_failure` throughout my changes